OWNERS_ALIASES

```diff
@@ -9,15 +9,11 @@ aliases:
     - ravisantoshgudimetla
   sig-scheduling:
     - bsalamat
-    - davidopp
-    - jayunit100
     - k82cn
     - resouer
-    - timothysc
```

Comment:
Please comment out for history vs. remove.

Reply:
It doesn't seem to be a common practice in the file. History of files are kept in Git.

Reply:
Other OWNERS files in the codebase "were" managed this way.  It looks like someone has gone through and ripped out the comments.  

That's fine. 
/lgtm